data/strings/strings.txt

```diff
@@ -22429,135 +22429,6 @@
     zh-Hans = 轮渡
     zh-Hant = 渡輪
 
-  [avoid_toll_roads_placepage]
```

Comment:
I think it makes sense to use these translations in the new string IDs like `[avoid_tolls]`. Because these translations had been in the repo for a long time and were proof-read by translators for sure, unlike most of the new automatic translations.

Reply:
Aren't they different?

Reply:
True, those strings are "Avoid toll roads", but the new ones are "Avoid tolls".
But "Avoid unpaved roads" and "Avoid ferry crossings" are the same.